src/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
```

Comment:
Your HTML code looks like in the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png) 🙌